### PR TITLE
Home Screen - Registration card (EXPOSUREAPP-6724)

### PR DESCRIPTION
**Pull request by jurajkusnier**
- Add register certificate card design on the home screen
- Implementation will be included in future PR
image

### CODE REVIEW

**Comment by fynngodau** [comment]
Quick question, since I don't have access to the mockups: will there also be a tab for displaying the certificate?

**Comment by jurajkusnier** [comment]
> Quick question, since I don't have access to the mockups: will there also be a tab for displaying the certificate?

The certificate will be displayed after clicking on another new home screen card.

**Comment by fynngodau** [comment]
@jurajkusnier Thanks for the info. I will open a wishlist item with my thoughts regarding tabs

**Review by chiljamgossow — DISMISSED** [review]
LGTM

File: Corona-Warn-App/src/main/res/layout/home_vaccination_registration_card.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            app:layout_constraintTop_toBottomOf="@+id/title" />
+
+        <androidx.appcompat.widget.AppCompatImageView
+            android:id="@+id/icon"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:importantForAccessibility="no"
+            android:layout_marginEnd="16dp"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="@+id/body"
```
Comment (axelherbstreith) [inline Corona-Warn-App/src/main/res/layout/home_vaccination_registration_card.xml:47]: please only use `@+id/` for `android:id`, else use `@id/ `

Comment (jurajkusnier) [reply Corona-Warn-App/src/main/res/layout/home_vaccination_registration_card.xml]: true, I reused an older card as template

**Review by axelherbstreith — APPROVED** [review]
LGTM

**Review by chiljamgossow — APPROVED** [review]

**Pull request merged**